### PR TITLE
Chore/create views

### PR DESCRIPTION
**Pull request by EmEhPa**
Just created views for almost every page. Some of them are redirected from the navbar some of them not but this will be fixed when the footer gets some attention and some other small fixes.

I also deleted the AboutView since we have a drop down and changed the name for the file ```frontpage```  to ```FrontPage```

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -20,17 +20,17 @@ import '@/assets/main.scss' // Ensure you import your custom main.scss
           <RouterLink to="/" class="navbar-item">Home</RouterLink>
 
           <div class="navbar-item has-dropdown is-hoverable">
-            <RouterLink to="/about" class="navbar-link">About</RouterLink>
+            <a class="navbar-link">About</a>
             <div class="navbar-dropdown">
-              <RouterLink to="/subpage1" class="navbar-item">The Team</RouterLink>
+              <RouterLink to="/theteam" class="navbar-item">The Team</RouterLink>
```
Comment (retrokiller543) [inline src/App.vue:25]: this could be `/team`

File: src/App.vue [hunk 2]
```diff
@@ -20,17 +20,17 @@ import '@/assets/main.scss' // Ensure you import your custom main.scss
...
               <hr class="navbar-divider" />
-              <RouterLink to="/subpage2" class="navbar-item">ThreatMapperAI</RouterLink>
+              <RouterLink to="/threattmapperai" class="navbar-item">ThreatMapperAI</RouterLink>
```
Comment (retrokiller543) [inline src/App.vue:27]: this could be `/ai`

File: src/router/index.ts [hunk 3]
```diff
@@ -30,56 +31,61 @@ const routes = [
...
-  component: ChatView
+    path: '/background',
+    name: 'background',
+    component: BackgroundView
   },
   {
-  path: '/profile',
-  name: 'profile',
-  component: ProfileView
+    path: '/threattmapperai',
```
Comment (retrokiller543) [inline src/router/index.ts:51]: see previous comment about paths

File: src/router/index.ts [hunk 4]
```diff
@@ -30,56 +31,61 @@ const routes = [
...
+    name: 'threattmapperai',
+    component: ThreatTMapperAIView
   },
   {
-  path: '/background',
-  name: 'background',
-  component: BackgroundView
+    path: '/theteam',
```
Comment (retrokiller543) [inline src/router/index.ts:56]: again see the previous comment about paths

File: src/App.vue [hunk 5]
```diff
@@ -20,17 +20,17 @@ import '@/assets/main.scss' // Ensure you import your custom main.scss
           <RouterLink to="/" class="navbar-item">Home</RouterLink>
 
           <div class="navbar-item has-dropdown is-hoverable">
-            <RouterLink to="/about" class="navbar-link">About</RouterLink>
+            <a class="navbar-link">About</a>
```
Comment (retrokiller543) [inline src/App.vue:23]: why is this a `<a/>` tag? can this not be a `<p/>` to avoid any html or vue errors in the future?

**Review by retrokiller543 — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 2 above)
Comment (EmEhPa) [reply src/App.vue:27]: Fixed

File: src/App.vue (hunk 1 above)
Comment (EmEhPa) [reply src/App.vue:25]: fixed

File: src/router/index.ts (hunk 3 above)
Comment (EmEhPa) [reply src/router/index.ts:51]: fixed

File: src/router/index.ts (hunk 4 above)
Comment (EmEhPa) [reply src/router/index.ts:56]: fixed

File: src/App.vue (hunk 5 above)
Comment (EmEhPa) [reply src/App.vue:23]: It works with </p>, lets go for it.

**Review by muaathaw — APPROVED** [review]
looks good

**Review by retrokiller543 — APPROVED** [review]

**Pull request merged**